backend/app.py

```diff
@@ -1,12 +1,24 @@
-from flask import Flask
-
-app = Flask(__name__)
+import requests
+from flask import Flask, jsonify
 
+def parse(forecast):
+    parsed = []
+    for period in forecast['periods']:
+        if period['name'] == 'Tonight':
+            parsed.append(period['name'])
+            parsed.append(period['detailedForecast'])
+        if period['name'] == 'Rest Of Today':
+            parsed.append(period['name'])
+            parsed.append(period['detailedForecast'])
+    return parsed
 
-@app.route("/", methods=["GET"])
-def welcome():
-    return "Hello World!"
+app = Flask(__name__)
 
+@app.route("/", methods = ["GET"])
+def weather(url):
+    response = requests.get(url)
```

Comment:
This should not be how the API is set up. You should have endpoints which then convert the passed in params into an API. For example, requesting `API_URL/weather/chicago` should respond with the weather in Chicago. I should never be passing the url I want the API to fetch - this would defeat the entire purpose of having a backend to begin with.